client/my-sites/promote-post-i2/components/search-bar/index.tsx

```diff
@@ -199,15 +247,20 @@ export default function SearchBar( props: Props ) {
 			<div className="promote-post-i2__search-bar-options">
 				{ mode === 'posts' && (
 					<>
+						{ isWooStore && isDesktop && (
+							<WooItemsFilter handleChangeFilter={ onChangeFilter } postType={ postType || '' } />
```

Comment:
Can't we use the same handler `onChangePostTypeFilter` as the default one here? 
Maybe it gets a bit messy because both will need to use the same value type `DropdownOption`. but right now the code is almost the same, so it would be nice to have a single dropdown.

Reply:
I'm refactoring this to use only one method :)

Reply:
Solved